opba-banking-rest-impl/src/test/resources/application.yml

```diff
@@ -28,14 +28,14 @@ api:
       start: 0
       max: 10
       query:
-        "SELECT id, uuid, name, bic, bank_code, word_similarity(:keyword, name) as sml
-        FROM opb_bank WHERE word_similarity(:keyword, name) >= 0.6
+        "SELECT id, uuid, name, bic, bank_code, active, word_similarity(:keyword, name) as sml
```

Comment:
Please replace query value in all YMLs (i.e. open-banking-embedded-starter)

Reply:
replaced